content/articles/2017-12-14-styler-1-0-0.Rmd

```diff
@@ -0,0 +1,230 @@
+---
+title: styler 1.0.0
+author: Kirill Müller, Lorenz Walthert
```

Comment:
@krlmlr I added you as an author.

Reply:
Thanks. I think you should be "first author" or "corresponding author", but that may not work for a blog post. Anyway, I'm not sure my contributions to this blog post qualify for an authorship.

Reply:
I just thought because of your involvement with styler (and no necessarily with this post), it's fair to list you, but if you prefer not to appear, I can remove you.

Reply:
Or put you second.

Reply:
Second author feels better to me.

Reply:
Ok.